### PR TITLE
Daniil/changed currency name and account prefix from UST to USDT

### PR DESCRIPTION
**Pull request by Daniil-Deriv**

### CODE REVIEW

**Comment by Daniil-Deriv** [comment]
also updated babel on Aaron's advice

File: src/botPage/view/View.js [hunk 1]
```diff
@@ -213,7 +213,9 @@ const updateTokenList = () => {
         }
 
         tokenList.forEach(tokenInfo => {
-            const prefix = isVirtual(tokenInfo) ? 'Virtual Account' : `${tokenInfo.loginInfo.currency} Account`;
+            const prefix = isVirtual(tokenInfo)
+                ? 'Virtual Account'
+                : `${tokenInfo.loginInfo.currency === 'UST' ? 'USDT' : tokenInfo.loginInfo.currency} Account`;
```
Comment (carol-binary) [inline src/botPage/view/View.js:218]: Nit. but can we handle this better by not nesting ternary operator? It'll make the code easier to understand as well.

File: package.json [hunk 2]
```diff
@@ -45,8 +45,9 @@
     "@babel/register": "^7.4.4",
     "@binary-com/binary-indicators": "^1.6.3",
     "app-module-path": "^2.2.0",
+    "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "^10.0.1",
-    "babel-jest": "^24.8.0",
+    "babel-jest": "^23.4.2",
```
Comment (reza-fs) [inline package.json:50]: (question) Why this change was needed?

Comment (reza-fs) [reply package.json:50]: Oh I see it was advised, but I was wondering why `babel-jest` is downgraded.

Comment (Daniil-Deriv) [reply package.json:50]: It is needed to pass Circle-ci tests. Aaron advised me to do so.

File: src/botPage/view/View.js (hunk 1 above)
Comment (Daniil-Deriv) [reply src/botPage/view/View.js]: changed ternary to 'if' statement

File: package.json (hunk 2 above)
Comment (Daniil-Deriv) [reply package.json:50]: Honestly I'm not sure. It was automatically changed after updating babel

**Review by carol-binary — APPROVED** [review]

**Review by nijil-binary — APPROVED** [review]
